src/utils/branch.ts

```diff
@@ -7,7 +7,9 @@ import { analyticsV2 } from '@/analytics';
 import * as ls from '@/storage';
 import { logger, RainbowError } from '@/logger';
 
-export const branchListener = async (handleOpenLinkingURL: (url: any) => void) => {
+const isEmpty = <T extends object>(obj: T | undefined): obj is undefined => !obj || Object.keys(obj).length === 0;
```

Comment:
could probably just use lodash isEmpty since we have that available.

Reply:
I don't know lodash so went to look at what it does, I don't think we need all that, and it "breaks" the types
https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11479